packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -577,7 +577,7 @@ export class AtomicSearchBox {
 
   private onSuggestionClick(item: SearchBoxSuggestionItem) {
     item.onSelect && item.onSelect();
-    this.clearSuggestions();
+    isSuggestionElement(item) && item.query && this.clearSuggestions();
```

Comment:
this makes sure that we don't blur the input when we click on something that doesn't update the input. I think this makes more sense.. since `clearSuggestions` is accessible from the implementer, elements can deal with it case by case if needed